pkg/cli/alpha/config-gen/testdata/default/expected.yaml

```diff
@@ -270,28 +125,28 @@ spec:
         control-plane: controller-manager
     spec:
       containers:
-        - command:
-            - /manager
-          args:
-            - "--metrics-bind-address=127.0.0.1:8080"
-            - "--leader-elect"
-          image: example/simple:latest
-          name: manager
-          resources:
-            limits:
-              cpu: 100m
-              memory: 30Mi
-            requests:
-              cpu: 100m
-              memory: 20Mi
-        - name: kube-rbac-proxy
-          image: gcr.io/kubebuilder/kube-rbac-proxy:v0.5.0
-          args:
-            - "--secure-listen-address=0.0.0.0:8443"
-            - "--upstream=http://127.0.0.1:8080/"
-            - "--logtostderr=true"
-            - "--v=10"
-          ports:
-            - containerPort: 8443
-              name: https
+      - command:
+        - /manager
+        args:
+        - "--metrics-bind-address=127.0.0.1:8080"
+        - "--leader-elect"
+        image: example/simple:latest
+        name: manager
+        resources:
+          limits:
+            cpu: 100m
+            memory: 30Mi
+          requests:
+            cpu: 100m
+            memory: 20Mi
+      - name: kube-rbac-proxy
+        image: gcr.io/kubebuilder/kube-rbac-proxy:v0.5.0
```

Comment:
Not part of this one
But we need in a follow up to upgrade the kube-proxy for 0.11.0

Reply:
It was changed when I ran the tests. Without it, the teats don't work.

Reply:
That is fine. This is == the scaffold produced in the config/ dir 
The version of the image is old we just need another task to update the config-gen and solve its tech-debts